grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\s+'
+    'begin': '^={1,6}\\p{Blank}+'
```

Comment:
Sorry but I don't get the issue. A trailing tab shouldn't break the syntax highlighting of a title.